src/tribler-core/tribler_core/modules/metadata_store/restapi/channels_endpoint.py

```diff
@@ -246,7 +245,7 @@ def get_channel_from_request(self, request):
         torrents_dir = None
         if parameters.get('torrents_dir', None):
             torrents_dir = parameters['torrents_dir']
```

Comment:
It is better to make `torrents_dir` a Path object itself, and then operate on that.

Reply:
Done